numalogic/registry/mlflow_registry.py

```diff
@@ -21,6 +21,8 @@
 from mlflow.exceptions import RestException
 from mlflow.protos.databricks_pb2 import ErrorCode, RESOURCE_DOES_NOT_EXIST
 from mlflow.tracking import MlflowClient
+from sklearn.base import BaseEstimator
+from torch import nn
```

Comment:
knitpick:

We already are doing the import in types.py. 
Is it possible for us to create pytorch type and scikit model type and then import it from there?

For fun I tried importing nn module directly vs from our types.py (very minute difference):
###### nn Module import
Start time : 2.470361608
Time difference : 1.842999999990269e-05

###### import from types.py
Start time : 2.470388254
Time difference : 6.133999999935469e-06

Reply:
So I tried that, but TypeVars don't work for `isinstance` check since they are not really actual class types.